### DISCUSSION TITLE
2023/24 season update

### DISCUSSION THREAD

**Discussion by makeitrainier** [kyleskom/NBA-Machine-Learning-Sports-Betting#340] [Q&A]
can someone please help me with updating this for the 2023/24 season. the xg predictions are coming through but not the neural network model predicitions.

**Comment by Atrone** [comment]
check out the website i made for this purpose:

https://github.com/Atrone/NBA-Machine-Learning-Sports-Betting

**Reply by Atrone** [reply]
if you want the details of the strategy, email me:

anthlift@gmail.com

**Comment by deperto** [comment]
How can we access it, it asks for user and pass?Thnx


Sent from Yahoo Mail for iPhone


On Tuesday, November 7, 2023, 16:52, Anthony Teixeira ***@***.***> wrote:




check out the website i made for this purpose:

https://github.com/Atrone/NBA-Machine-Learning-Sports-Betting

**Reply by Atrone** [reply]
yeah bro email me: anthlift@gmail.com

**Reply by STRATZ-Ken** [reply]
Here we go.  Pay him for access.  Sigh, gotta ban this guy. @kyleskom

**Reply by Emillzz45** [reply]
Very unfortunate if so :/

**Reply by Atrone** [reply]
How is that unfortunate? There's no free lunch guys. I added value to the tool, it's been giving me a lot of finanical help, and I want to share it... for a goddamn price! What do you think this is?

**Reply by Atrone** [reply]
If Kyle wants in on the PRODUCT (yes, that's what it should be), then he can respond to my DM and we can profit share. It's very simple.

**Reply by chriseling** [reply]
This is open source

**Reply by Atrone** [reply]
Ok? Then he can feel free to follow the exact instructions in my repo and make his own website. Sure. If he wants my strategy around the algorithm however, I have made that proprietary.

**Reply by STRATZ-Ken** [reply]
AKA is there is no formula.  The base idea is wrong.  The base results from the model are wrong.  But hey keep putting lipstick on a pig.

**Reply by Atrone** [reply]
? base idea is wrong? It's an LSTM with tools trying to perform statistical arbitrage. That's the best algorithm we have for time series data, and I use the same "base idea" for my stock trading algorithm that has performed marvelously. What have you done?

**Reply by STRATZ-Ken** [reply]
Sigh.  Again, I say, please ban this guy.

This is the classic, my betting model works so good, I have to resell it because I make too much money from it and I want to take 10-20$ from people because that 10$ is easier to get then click "submit bet" on FanDuel.  So dumb.

**Reply by Atrone** [reply]
checkmate

**Reply by Emillzz45** [reply]
Na mate. Just weird activity is all

**Reply by Atrone** [reply]
It's 100% weird what's happening right now. We've got a good (and this is based on real bets) model for NBA betting, and it is hard to run, understand, and trust. 

I'm offering to solve these problems. I've DMed Kyle about this, and I'm awaiting a response. I'm not hiding anything. I told OP to check this thread and make sure he's interested in what i'm offering

**Comment by Baaios** [comment]
He is going to ask for you to pay him. Don't. This is Kyle's work, this guy
doesn't take a part in it.

**Reply by Atrone** [reply]
Ok? How else will he have convienent access to this tool? I don't understand your position here. I forked the repo and added a lot of value to it. Why not sell it?

**Reply by STRATZ-Ken** [reply]
Do you understand what the heck the point of github is? Hey guys, React is open source code.  I'm just going to fork it and sell it as my own!

You didnt add any value to it. You just made a way for the incorrect bets, and wrong return values to be pretty and designed.  Did you collect Injury reports? Did you collect trade reports?  No. No. No.  The entire way that this project is laid out is INCORRECT and WRONG for betting.  However, this repo gives you the IDEA on how to use NNs.  Your just scamming people for money.

Your bringing nothing to this community but a toxic nature.

**Reply by chriseling** [reply]
There's very little value in the .bat file you made.  Almost no value here in the fork.

**Reply by Atrone** [reply]
The fork is open source. He can literally clone and run the entry script and get today's bets. awesome. the strategy i've analyzed from the hundreds of live bets however is proprietary.

**Reply by Atrone** [reply]
also @chriseling , there's a ton of value in the .bat file for people who aren't software engineers. if the product is not easily runnable, it's worthless to them.

**Reply by Atrone** [reply]
i sent a SC

**Reply by Atrone** [reply]
![Screenshot_20231107-112248](https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/assets/35811311/a400254d-6876-48c7-a592-41b44be99691)
![Screenshot_20231107-112237](https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/assets/35811311/9967afaa-95df-4d6e-8ff2-69e6ab17667d)
![Screenshot_20231107-112239](https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/assets/35811311/35983a36-52a5-4c1c-b434-b4b1b9068ca5)
![Screenshot_20231107-112246](https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/assets/35811311/a10756b7-94e3-459b-8555-406f456da17a)

400$ bets are 2022, mixed value is 2023, last night i lost every bet except minnesota.

**Reply by Atrone** [reply]
@Gxent why not bet on +EV? That's the correct idea.

**Reply by Atrone** [reply]
once again, misunderstanding the product with negative consequences.

**Reply by Atrone** [reply]
another potential insight: later season games give more profit. maybe if there was some incentive to improve the product it would improve faster 🤔

**Reply by Atrone** [reply]
@Gxent Sure. 1 sec. I used FanDuel

**Reply by Atrone** [reply]
![Screenshot_20231107-113538](https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/assets/35811311/1100a78f-18f8-455a-b079-2063c86636ea)
![Screenshot_20231107-113541](https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/assets/35811311/859b9785-cee0-418b-a579-7f066fb8c385)
![Screenshot_20231107-113533](https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/assets/35811311/bde181f0-9b90-4eca-bed9-490e3d82c797)

Here are some older bets from last season. Not personally comfortable giving bet Ids for my new betting app, as I've been hacked before.

**Reply by Atrone** [reply]
@Gxent

**Reply by Atrone** [reply]
lol. as soon as the proof comes out everyone leaves... anyways. i got to work. pce! ban me if you want lol i have my money

**Reply by Emillzz45** [reply]
I guess 9 bet slips and shit written on a notepad makes you the man. Enjoy the rest of your "work" day.

**Reply by Atrone** [reply]
thanks you too

**Reply by albeng8** [reply]
not really sure what value @Atrone is bringing ? Whatever he is doing can be done in a day. Most of his time should be spend on finetuning the model to get better outcome.  Maybe if he implement higher win%  xgboost model or doing some ML for spreads , i would consider paying.

**Reply by Atrone** [reply]
Thanks for asking rather than simply yelling at me.

I'm proposing that I bring operations to the project. Sensitivity analysis (how much for each EV), different strategies (low risk, high risk, etc), some forward test (not backtest) data to help people visualize the strategy, as well as the fully automated system (but that is not the sole value, if you want just that, it's 100% free on my fork. just clone it).

There's multiple people confused as to what this repo is or how to use it, but people are interested.

I'm not saying to shout this repo from the rooftops for 20$ a season, not even close. I'm just suggesting that if you're not a software engineer, or if you're simply lazy, I can come in and spoon feed you software that helps bet. That's it. These people yelling are just absurd.

**Reply by Atrone** [reply]
@albeng8

**Reply by albeng8** [reply]
i think people are mad because people usually dont ask for money when they contribute / add value to the project. I rather just spend my time cleaning up data / finding new features to add to improve the % of the model. For example , some of the closing lines for the totals are wrong if you compare historical data pulled from odds api.

**Reply by Atrone** [reply]
yeah, do that then. i'd be happy to help if you need. they can be mad, that's fair

**Comment by Atrone** [comment]
400$ bets are 2022, mixed value bets are 2023.

**Reply by Atrone** [reply]
1 sec, phone didnt attach image from gmail

**Comment by Atrone** [comment]
mhm
